images/linux/scripts/installers/google-chrome.sh

```diff
@@ -25,3 +25,52 @@ fi
 # Document what was added to the image
 echo "Lastly, documenting what we added to the metadata file"
 DocumentInstalledItem "Google Chrome ($(google-chrome --version))"
+
+CHROME_VERSION=$(google-chrome --version | grep -Eo "([0-9]+\.?){4}" | cut -d "." -f 1,2,3)
```

Comment:
Can be simplified to :

```suggestion
CHROME_VERSION=$(google-chrome --product-version)
CHROME_VERSION=${CHROME_VERSION%.*}
```

Reply:
Looks good. Thanks for your suggestion